contracts/networks/rinkarby.json

```diff
@@ -0,0 +1,17 @@
+{
+    "parameters": {},
+    "addresses": {
+        "verificationGateway": "0x216b5cA8aB30ce0f4Ba05C4Dbc92E0194a48850c",
+        "blsLibrary": "0xA86c40ad4b04E0228BB89B672bb32214Cb9bf02f",
+        "blsExpander": "0x4312117E6aD174504c662c320241975018FC3a1B",
+        "testToken": "0x8CD755F93E56C90E3d5803c036771CFCc12B7c53",
+        "rewardToken": "0x8CD755F93E56C90E3d5803c036771CFCc12B7c53"
+    },
+    "auxiliary": {
+        "chainid": 421611,
+        "domain": "TODO keccak256(abi.encodePacked(uint32(0xfeedbee5)))",
+        "genesisEth1Block": 123456,
+        "deployedBy": "",
+        "version": "c1d1f197ba9b68284cbc4ec145e286a4e84c0eaf"
+    }
+}
```

Comment:
@jzaki @voltrevo I would like to move the contract addresses and other deployment metadata out of the per directory env files and into a file modeled after `genesis.json` deployment metadata file we used in Hubble (https://github.com/thehubbleproject/hubble-contracts/blob/master/docs/setup/LOCAL_DEVELOPMENT.md#deploy).

The number one issue me and @kautukkundan ran into while getting setup was configuring these correctly across the aggregator and extension. Imagine others will as well. This has the advantage of having all the contract and onchain deployment info in one spot, at the cost of having to pass in a path to this config file when running the aggregator or extension. I think this change is worth making now, but open to other considerations/ideas.

Reply:
Sounds good to me. Definitely need some kind of combined solution, and I like moving to a json file too.